### PR TITLE
Add add and update features

### PR DESCRIPTION
**Pull request by nachosala89**
- Implemented a feature for adding a new task.
- Implemented a feature for editing task descriptions.
- Implemented features for deleting a task and clear all completed.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @nachosala89 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/showItems.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+};
+
+export const addTask = (tasksList, addIn) => {
+  if (addIn.value !== '') {
+    const description = addIn.value;
+    listAddTask(tasksList, description);
+    addIn.value = '';
+    const listContainer = document.querySelector('ul');
+    showTask(listContainer, tasksList, tasksList.length - 1);
+  }
```
Comment (mikenath223) [inline src/showItems.js:85]: - [ ] Kindly include localStorage functionality when adding and deleting a to-do item. After adding a to-do item and refreshing the page the to-do item should still appear on the page also when the item is deleted it should also be removed from localStorage.

File: src/showItems.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    const icon = text.nextElementSibling;
+    icon.classList.add('fa-ellipsis-v');
+    icon.classList.remove('fa-trash-alt');
+  });
+
+  const check = document.querySelector(`.item-${index} input`);
+  check.addEventListener('change', () => { updateStatus(tasksList, index); });
+
+  const rmBtn = document.querySelector(`.item-${index} i`);
+  // eslint-disable-next-line no-use-before-define
```
Comment (mikenath223) [inline src/showItems.js]: - [ ] Please remove this eslint disable line, it's best to fix the linting issue in the code.

File: src/showItems.js (hunk 1 above)
Comment (nachosala89) [reply src/showItems.js:85]: All the calls to localStorage for saving the values are made in the module listManager.js, and in this function here is called through the function listAddTask(tasksList, description). At least in my local environment, the functionality works well.

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**